icu4c/source/data/mappings/gb18030.ucm

```diff
@@ -30010,16 +30032,16 @@ CHARMAP
 <UE78A> \xA6\xBE |0
 <UE78B> \xA6\xBF |0
 <UE78C> \xA6\xC0 |0
-<UE78D> \xA6\xD9 |0
-<UE78E> \xA6\xDA |0
-<UE78F> \xA6\xDB |0
-<UE790> \xA6\xDC |0
-<UE791> \xA6\xDD |0
-<UE792> \xA6\xDE |0
-<UE793> \xA6\xDF |0
-<UE794> \xA6\xEC |0
-<UE795> \xA6\xED |0
-<UE796> \xA6\xF3 |0
+<UE78D> \x84\x31\x82\x36 |0
```

Comment:
As discussed in https://www.unicode.org/L2/L2023/23003r-gb18030-recommendations.pdf, these mapping changes are bad for existing data, both in Unicode and in GB 18030.

AFAIU from discussion at UTC-175, it should be acceptable to use backwards compatible mappings like this:
```
<UE78D> \xA6\xD9 |1
<UFE10> \xA6\xD9 |0
<UFE10> \x84\x31\x82\x36 |3
```
FYI @kenlunde

Reply:
I put together improved mappings for all of the 18 characters and their PUA siblings:
```
<UFE10> \xA6\xD9 |0
<UE78D> \xA6\xD9 |1
<UFE10> \x84\x31\x82\x36 |3

<UFE12> \xA6\xDA |0
<UE78E> \xA6\xDA |1
<UFE12> \x84\x31\x82\x38 |3

<UFE11> \xA6\xDB |0
<UE78F> \xA6\xDB |1
<UFE11> \x84\x31\x82\x37 |3

<UFE13> \xA6\xDC |0
<UE790> \xA6\xDC |1
<UFE13> \x84\x31\x82\x39 |3

<UFE14> \xA6\xDD |0
<UE791> \xA6\xDD |1
<UFE14> \x84\x31\x83\x30 |3

<UFE15> \xA6\xDE |0
<UE792> \xA6\xDE |1
<UFE15> \x84\x31\x83\x31 |3

<UFE16> \xA6\xDF |0
<UE793> \xA6\xDF |1
<UFE16> \x84\x31\x83\x32 |3

<UFE17> \xA6\xEC |0
<UE794> \xA6\xEC |1
<UFE17> \x84\x31\x83\x33 |3

<UFE18> \xA6\xED |0
<UE795> \xA6\xED |1
<UFE18> \x84\x31\x83\x34 |3

<UFE19> \xA6\xF3 |0
<UE796> \xA6\xF3 |1
<UFE19> \x84\x31\x83\x35 |3

<U9FB4> \xFE\x59 |0
<UE81E> \xFE\x59 |1
<U9FB4> \x82\x35\x90\x37 |3

<U9FB5> \xFE\x61 |0
<UE826> \xFE\x61 |1
<U9FB5> \x82\x35\x90\x38 |3

<U9FB6> \xFE\x66 |0
<UE82B> \xFE\x66 |1
<U9FB6> \x82\x35\x90\x39 |3

<U9FB7> \xFE\x67 |0
<UE82C> \xFE\x67 |1
<U9FB7> \x82\x35\x91\x30 |3

<U9FB8> \xFE\x6D |0
<UE832> \xFE\x6D |1
<U9FB8> \x82\x35\x91\x31 |3

<U9FB9> \xFE\x7E |0
<UE843> \xFE\x7E |1
<U9FB9> \x82\x35\x91\x32 |3

<U9FBA> \xFE\x90 |0
<UE854> \xFE\x90 |1
<U9FBA> \x82\x35\x91\x33 |3

<U9FBB> \xFE\xA0 |0
<UE864> \xFE\xA0 |1
<U9FBB> \x82\x35\x91\x34 |3
```

Reply:
Many thanks, this makes sense! My original mapping changes were based on some early internal discussions at Apple, not the recommendations doc or UTC-175 discussion.